docker/Dockerfile

```diff
@@ -14,9 +14,12 @@ ARG bazel_jobs=""
 ARG git_clone="true"
 
 RUN apt-get update
-RUN apt-get install -y git sudo python-pip python3-pip
+RUN apt-get install -y git sudo python3-pip
 RUN git clone https://github.com/pytorch/pytorch
 
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+RUN sudo apt-get install -y -q
```

Comment:
Could you elaborate a bit on what this line is trying to do?

Reply:
I was getting debian front end config issues, it was going into interactive mode and expecting user response. 
I see you aleady have this for the wheel script (https://github.com/pytorch/xla/blob/master/scripts/build_torch_wheels.sh#L9) , but still it was going to interactive mode

> DEBIAN_FRONTEND=noninteractive

When I looked around, this is where I found the above solution
https://github.com/moby/moby/issues/27988#issuecomment-462809153